src/Port.js

```diff
@@ -0,0 +1,20 @@
+function Port(name) {
+    this.name = name;
+    this.ships = [];
+};
+
+Port.prototype.addShip = function(ship) {
+    const ships = this.ships;
+    
+    ships.push(ship); 
+};
+
+Port.prototype.removeShip = function(ship) {
+
+    const ships = this.ships;
+    
+    ships.pop(ship);
+}
+
```

Comment:
You dont need to create a new variable from `this.ships`, you can just refer to it directly:
```
Port.prototype.addShip = function (ship) {
    this.ships.push(ship); 
};

Port.prototype.removeShip = function (ship) {
    this.ships.pop(ship);
}
```